backend/test.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+
+if os.path.exists('.env'):
+    print('Importing environment from .env file')
+    for line in open('.env'):
+        var = line.strip().split('=')
+        if len(var) == 2:
+            os.environ[var[0]] = var[1]
+
+test2= os.environ.get("FLASK_CONFIG")
```

Comment:
Fix the formatting with the `=` signs so that there is a space on each side

Reply:
Yes! I could upload a new video later in slack to demonstrate the process.